### PR TITLE
Migrate Sentry packages to v8

### PR DESCRIPTION
**Pull request by goplayoutside3**
## Package
app-project
lib-classifier

## Linked Issue and/or Talk Post
Includes https://github.com/zooniverse/front-end-monorepo/pull/6372
Reverts https://github.com/zooniverse/front-end-monorepo/pull/4387

## Describe your changes
- Migrate sentry/browser and sentry/nextjs to version 8.
- Remove Sentry initialization in app-project's _document.js file. This was a specific tracking case for FrontDoor behavior 2 years ago and has since resolved.

## How to Review

- This is deployed to the fe-branch deploy for testing. View any project such as https://fe-project-branch.preview.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify and you'll see the Sentry init log in the console: `Initialising Sentry (browser runtime):"https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691" "branch""7351b3d6df09d1c95e3704bfa5582150d6668341"` unless an adblocker affects it.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)

## Maintenance
- [x] If not from dependabot, the PR creator has described the update (major, minor, or patch version, changelog)

### CODE REVIEW

File: packages/app-project/server/server.js [hunk 1]
```diff
@@ -27,16 +26,13 @@ const handle = app.getRequestHandler()
 
 app.prepare().then(() => {
   const server = express()
-
-  server.use(Sentry.Handlers.requestHandler())
```
Comment (goplayoutside3) [inline packages/app-project/server/server.js:31]: Sentry.Handlers isn't needed anymore

File: packages/app-project/instrumentation.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export async function register() {
```
Comment (goplayoutside3) [inline packages/app-project/instrumentation.js:1]: See https://nextjs.org/docs/app/api-reference/file-conventions/instrumentation

File: packages/app-project/next.config.js [hunk 3]
```diff
@@ -119,7 +116,12 @@ const nextConfig = {
 module.exports = million.next(
   withSentryConfig(nextConfig, {
     org: 'zooniverse-27',
-    project: 'fem-app-project'
+    project: 'fem-app-project',
+    sourcemaps: {
+      disable: true,
+    },
+    telemetry: false,
+    authToken: process.env.SENTRY_AUTH_TOKEN
```
Comment (mcbouslog) [inline packages/app-project/next.config.js:124]: It looks like this was not required previously, but is with these changes, it looks like per https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#extend-your-nextjs-configuration .

**Review by mcbouslog — APPROVED** [review]
Noted sentry console logs and network activity as expected when testing https://fe-project-branch.preview.zooniverse.org/ 👍

**Pull request merged**